distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: Implementation of the user-defined currency
 - Feature: Add OpenGL drawing engine.
```

Comment:
Shouldn't the OpenGL engine feature be on top of the changelog, as it seems to be an even bigger change then the implementation of the user-defined currency?